static/katex.less

```diff
@@ -15,7 +15,7 @@
 }
 
 .katex {
-    font: normal 1.21em KaTeX_Main;
+    font: normal 1.21em KaTeX_Main, Times New Roman;
```

Comment:
We should definitely have `serif` as a last item in this list, in case a user doesn't have Times New Roman installed. I'm also thinking whether we should prefer [DejaVu](http://dejavu-fonts.org/wiki/Main_Page) Serif over Times New Roman (i.e. list both but DejaVu first). I think that DejaVu might contain a wider range of glyphs, and that it would be easier for site admins to legally ship a webfont version of it to ensure consistent rendering. To my knowledge Times New Roman is still proprietary, plus the version on Mac is different from the one on Windows.